scripts/setup-linux.sh

```diff
@@ -2,3 +2,12 @@
 sudo apt -y update
 sudo apt-get install -y protobuf-compiler libseccomp-dev
 
+echo "setting up rust"
+sudo rustup toolchain install 1.74 --component clippy --component rustfmt --no-self-update
+sudo rustup default 1.74
+sudo rustup target add wasm32-wasi && rustup target add wasm32-unknown-unknown
```

Comment:
Is sudo needed for `rustup target add`? If so:
```suggestion
sudo rustup target add wasm32-wasi && sudo rustup target add wasm32-unknown-unknown
```

Reply:
good catch. thank you. I am using the pattern we had earlier in script (to use sudo).